lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Balance.hs

```diff
@@ -1170,8 +1170,7 @@ selectAssets era (ProtocolParameters pp) utxoAssumptions outs redeemers
 data ChangeAddressGen s = ChangeAddressGen
     { getChangeAddressGen :: s -> (Address, s)
 
-    -- | Returns the longest address that the wallet can generate for a given
-    --   key.
+    -- | Returns a dummy change address of the maximum possible length.
```

Comment:
Could be good to write: "maximum possible length **'getChangeAddressGen' can generate**" or something similar

Reply:
@Anviking good point!

I've added a statement of this property in the following commit: 212301f0315e5c15606316f82f5e7cdcbf8f4ee0.